.github/workflows/system_tests.yaml

```diff
@@ -27,6 +27,10 @@ jobs:
           name: rcc
           path: C:\
 
+      # We need to opt-out of telemetry. Not sure why this is started though, since we use GNU for
+      # compilation, not MSVC. Telemetry will interfere with `examples/termination/main.rs`.
+      - shell: pwsh
+        run:  Get-ChildItem -Filter vctip.exe -Recurse "C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC\\Tools\\MSVC" | rm
```

Comment:
Is the problem that vctip.exe is started at some point which messes up our process tree?

Reply:
Yes, it was probably always the case, but it failed yesterday, after I updated the cargo.lock: https://github.com/elabit/robotmk/actions/runs/10996567682/attempts/1

Reply:
Hrmpf. Ok, then let's hope it's never started before we remove it. But I guess we just have to try out and see. Please expand the comment above a bit to explain that apparently, vctip.exe might be started randomly at some point during the test run, messing up the process tree.

Reply:
I think it is started by the `cargo --examples termination` invocation, otherwise it would not interfere with our test. I added the information I have.